### ISSUE TITLE
Print Register of Shareholders

### ISSUE THREAD

**Issue by winterstein** [good-loop/commercial#22]
Seedrs lawyers would like us to keep a printed copy, so why not?
Let's do that once we have the investment sorted.

NB: We also need to put Amy's address into the docs
https://drive.google.com/drive/u/0/folders/0BzLX1qrCSys9Q3pERlF6ZTd1dW8

**Comment by afwilli** [comment]
Agreed! My details are now on all sheets so just need to print after signing.
